🐛 [Bug]: pnpm dev Local startup error report: Failed to resolve entry for package "@opentiny/vue".

### Version

latest

### Vue Version

latest

### Link to minimal reproduction

执行 `pnpm dev` 本地启动项目，提示以下错误：

```
✘ [ERROR] Failed to resolve entry for package "@opentiny/vue". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]
```



### Step to reproduce

项目根目录执行 `pnpm dev` 命令

### What is expected

不报错，项目启动之后显示以下信息：
```
  VITE v4.2.1  ready in 1487 ms

  ➜  Local:   http://localhost:6175/
  ➜  Inspect: http://localhost:6175/__inspect/
```
这样就可以一目了然看到端口号。




### What is actually happening

现在显示了很多报错信息，导致看不到端口好，然后又没有自动打开浏览器，开发者会以为启动失败。

### Any additional comments (optional)

_No response_

--- a/packages/vue/package.json
+++ b/packages/vue/package.json
@@ -3,7 +3,6 @@
   "private": true,
   "version": "3.7.0",
   "description": "opentiny vue components",
-  "main": "index.ts",
   "type": "module",
   "files": [
     "index.js",

